cmd/manager/main.go

```diff
@@ -57,89 +55,87 @@ func parseSyncPeriod() (*time.Duration, error) {
 }
 
 func main() {
-	logf.SetLogger(zap.New(zap.UseDevMode(false)))
-	log := logf.Log.WithName("entrypoint")
-
 	// Get version of KUDO
-	log.Info(fmt.Sprintf("KUDO Version: %s", fmt.Sprintf("%#v", version.Get())))
+	log.Printf("KUDO Version: %s", fmt.Sprintf("%#v", version.Get()))
```

Comment:
Hmm, isn't this equivalent to:
```suggestion
	log.Printf("KUDO Version: %#v", version.Get())
```